pkg/oci/load_balancer.go

```diff
@@ -293,7 +287,18 @@ func (cp *CloudProvider) createLoadBalancer(ctx context.Context, spec *LBSpec) (
 	}
 
 	logger.With("loadBalancerID", *lb.Id).Info("Load balancer created")
-	return loadBalancerToStatus(lb)
+	status, err := loadBalancerToStatus(lb)
```

Comment:
I moved adding seclists to after we have created the loadbalancer. If creation fails for some reason the rules will not be added. Could there be unforseen issues with this?